bedrock/settings/static_media.py

```diff
@@ -1165,6 +1165,14 @@
         ),
         'output_filename': 'js/firefox_developer_firstrun-bundle.js',
     },
+    # will be needed when videos are ready (shortly after 6/2?)
```

Comment:
Not sure I quite follow why this is commented out - isn't this bundle needed for locales who don't have the `l10n_has_tag('spring_release')` tag?